db/schema.rb

```diff
@@ -10,9 +10,29 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 0) do
+ActiveRecord::Schema.define(version: 20170725213602) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "spells", force: :cascade do |t|
+    t.string "name", null: false
+    t.string "school", null: false
+    t.string "casting_time", null: false
+    t.string "spell_resistance"
+    t.string "saving_throw"
+    t.boolean "dismissible", default: false
+    t.string "description", null: false
+    t.string "description_short"
+    t.string "source"
+    t.boolean "verbal", default: false
```

Comment:
I almost feel `spell_requirements` could be something like `[:verbal, :somatic, :focus]` and those are the actual physical/preparations for the spell